codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/util/ShapeUtil.java

```diff
@@ -27,15 +27,15 @@
 
 public final class ShapeUtil {
     public static final StringShape STRING_SHAPE = StringShape.builder()
-            .id("smithy.api#String")
+            .id("smithy.api#PrimitiveString")
```

Comment:
I know we talked about these offline, and since these tests work I guess this is fine, but I couldn't find any docs about the existence of this type

Reply:
OK I finally found it. It's in the _prelude_ model, not the _preamble_ model, I was searching the wrong thing - https://smithy.io/2.0/spec/model.html#prelude